src/components/stepper/StepFinal.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import { Typography } from "@mui/material";
+
```

Comment:
A lot of unnecessary vertical whitespace (new lines) on this file and others. One new line between sections is enough.

Reply:
Done